README.md

```diff
@@ -7,8 +7,6 @@
 
 ----
 
-Check out [vella.ai/auth](https://vella.ai/auth/) for a demo app using this library for local-only authentication with WebAuthn and local encryption.
```

Comment:
Pleas update link to https://next.vella.ai/auth instead of removing

Reply:
ahh glad to know it's not gone :)

Reply:
great news! done ✅